fontforgeexe/cvstroke.c

```diff
@@ -114,23 +114,23 @@ static void CVStrokeIt(void *_cv, StrokeInfo *si, int justapply) {
 		cur = SplineSetStroke(spl,si,cv->b.layerheads[cv->b.drawmode]->order2);
```

Comment:
Not quite sure if this has the right effect, but maybe we could insert a 

```c
SCClearSelPt(cv->b.sc);
```
somewhere around here.

Reply:
I don't think the stroke code itself cares at all about selection.

Reply:
Yeah, but that's what's causing the valgrind/memory issues.

In this function you've just freed the SPL but there's still references to a selected point in that SPL, which is then being dereferenced in the charview (on cv->p)